content/docs/user-guide/assets/pipeline/metadata.md

```diff
@@ -0,0 +1,93 @@
+---
+linkTitle: Metadata Asset Relocation
+title: Metadata Asset Relocation
+description: In Open 3D Engine (O3DE), Metadata Asset Relocation allows files to be moved and renamed freely without breaking existing references by storing a UUID in a side-car file (.meta).
+weight: 600
+toc: true
+---
+

```

Comment:
I would love to see something like:

```
{{< note >}}
This feature is a work-in-progress. Don't depend on it yet; it may go through significant changes.
{{< /note >}}

Reply:
I applied the rewording to the first paragraph to mention this is experimental.  Should this note still be added as well?